src/client/src/OpenFin/apps/Launcher/NlpSuggestions/executions/TradeExecution/state.ts

```diff
@@ -15,92 +13,17 @@ import {
 
 import { Direction } from "@/generated/TradingGateway"
 import { getCurrencyPair$ } from "@/services/currencyPairs"
-import {
-  execute$,
-  ExecutionStatus,
-  ExecutionTrade,
-} from "@/services/executions"
+import { execute$, ExecutionStatus } from "@/services/executions"
 import { getPrice$ } from "@/services/prices"
 
-import { nlpIntent$, NlpIntentType } from "../../services/nlpService"
+import { nlpIntent$, NlpIntentType } from "../../../services/nlpService"
```

Comment:
@/services/..

Reply:
as we discussed this is a service folder in the Launcher/  ~different to the other ones